pgs_circle_sel.sql.in

```diff
@@ -0,0 +1,109 @@
+CREATE FUNCTION spoint_contained_by_circle_sel(internal, oid, internal, integer)
```

Comment:
That's the general signature of a selectivity estimator function. It's never called directly by the user (which wouldn't be possible anyway with `internal`), there is basically nothing to document about the individual parameters except that they need to be there so PostgreSQL can call the function. There is a comment on the function:

```
COMMENT ON FUNCTION spoint_contained_by_circle_sel(internal, oid, internal, integer) IS
   'selectivity estimator function for spoint_contained_by_circle';
```

We could link to the PG docs on how selectivity estimators work, but it would entirely generic PG documentation, nothing specific to pgsphere.

Should we?